wurst/systems/units/hostiles/trolls/Assassin.wurst

```diff
@@ -1,4 +1,5 @@
 package Assassin
+import ID
```

Comment:
Why did you add this to all entity classes? Seems like it isn't being used.

Reply:
@jonathanmclaus it is being used. You probably have the old war3map.j.